composer.json

```diff
@@ -4,12 +4,12 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
         "ext-json": "*",
-        "dvsa/mot-cpms-common": "^1.0",
+        "dvsa/mot-cpms-common": "dev-feature-bl-17035-php-82-update",
```

Comment:
Create a tag in dependency repos and use it here. You can create a tag before merging to master, and after merging you can correct it if needed.

Reply:
Tag version created and pulled to the composer